src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -87,7 +95,8 @@ private async System.Threading.Tasks.Task InitOnUIThreadAsync()
                     serviceProvider.GetMefService<IProjectToLanguageMapper>());
 
                 this.suppressedIssuesFileSynchronizer = await this.GetMefServiceAsync<ISuppressedIssuesFileSynchronizer>();
-                suppressedIssuesFileSynchronizer.UpdateFileStorage();
+                suppressedIssuesFileSynchronizer.UpdateFileStorageAsync().Forget(); // don't wait for it to finish
```

Comment:
The `update` method is now async and switches to the background thread. This method isn't waiting for it, so it continues on the UI thread immediately.